pkg/csv/csv_test.go

```diff
@@ -23,3 +23,7 @@ func TestCsvWriter(t *testing.T) {
 		t.Fatal(w.Error())
 	}
 }
+
+func TestCsvReader(t *testing.T) {
+
```

Comment:
Do you want to fill something in this test function later ?

Reply:
sry, have been removed it :) maybe I will add it in the next pr